packages/localize/src/tools/src/extract/translation_files/xliff1_translation_serializer.ts

```diff
@@ -24,9 +25,12 @@ const LEGACY_XLIFF_MESSAGE_LENGTH = 40;
  * @see Xliff1TranslationParser
  */
 export class Xliff1TranslationSerializer implements TranslationSerializer {
+  static validFormatOptions = new Map<string, string[]>([['xml:space', ['preserve']]]);
   constructor(
-      private sourceLocale: string, private basePath: AbsoluteFsPath,
-      private useLegacyIds: boolean) {}
+      private sourceLocale: string, private basePath: AbsoluteFsPath, private useLegacyIds: boolean,
+      private formatOptions: Record<string, string>) {
```

Comment:
We can probably use `FormatOptions` type here:
```suggestion
      private formatOptions: FormatOptions) {
```